### DISCUSSION TITLE
suggest a translation

### DISCUSSION THREAD

**Discussion by Fatichti** [upptime/upptime#650] [Ideas]
It would be great to be able to offer a **French version** for example for French applications and services.

If necessary, I can offer to do the translations but I would need help to implement them in the repository

Thx

**Comment by AnandChowdhary** [comment]
Thanks for the offer! We allow i18n of the site (https://upptime.js.org/docs/configuration/#internationalization) but the default build doesn't include multiple languages. A good idea to have!

**Reply by KaKi87** [reply]
Hello,

How can I add multiple languages to my Upptime instance ?

Thanks

**Reply by AnandChowdhary** [reply]
Unfortunately, multiple languages is currently unsupported.

**Reply by KaKi87** [reply]
Well, supporting multiple languages is the definition of i18n :sweat_smile:

**Reply by AnandChowdhary** [reply]
You're right, and I'll be happy to approve a PR that adds support! :) We currently allow you to use any single language, but perhaps a nice language switcher would be a good feature addition.

**Comment by bhdicaire** [comment]
Hello @Fatichti 

I just completed the translation of all i18N items in French: [Upptimerc.yml]( https://github.com/horticompetences-ca/status-page/blob/main/.upptimerc.yml)

**Reply by alphaleadership** [reply]
in private
